flaggr/src/main/java/com/comuto/flag/strategy/Strategy.java

```diff
@@ -16,5 +16,5 @@
      * @param context The context used to do operation
      * @return true if the flag is activated, false otherwise
      */
-    boolean isFlagActivated(Flag flag, FlagContextInterface context);
```

Comment:
If we are exposing it, we might want to change the name of the method as well since this one indicates that the possible outcome is either true/false.